src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyCacheableRequest.java

```diff
@@ -491,101 +297,126 @@ private boolean storeRequest(
+                    factory.writer.doReset(accept, acceptRouteId, acceptInitialId, reset.traceId());
+                    cleanupRequestIfNecessary();
                 }
             });
 
-            final String authorizationToken = requestGroup.getRecentAuthorizationToken();
-            if (authorizationToken != null)
-            {
-                builder.item(item -> item.name(AUTHORIZATION).value(authorizationToken));
-            }
-        };
+        factory.writer.doHttpRequest(connect,
+                                     connectRouteId,
+                                     connectInitialId,
+                                     signal.traceId(),
+                                     mutator);
+        cleanupRequestSlotIfNecessary();
     }
 
-    private void onResponseSignal(
+    private void onResponseSignalCacheEntryNotModified(
         SignalFW signal)
     {
-        final long signalId = signal.signalId();
-
-        if (signalId == CACHE_ENTRY_UPDATED_SIGNAL)
-        {
-            handleCacheUpdateSignal(signal);
-        }
-        else if (signalId == REQUEST_EXPIRED_SIGNAL)
-        {
-            DefaultCacheEntry cacheEntry = factory.defaultCache.get(requestHash);
-            factory.defaultCache.send304(cacheEntry,
-                                         getHeader(getRequestHeaders(), PREFER),
-                                         acceptReply,
-                                         acceptRouteId,
-                                         acceptReplyId);
-            cleanupRequestIfNecessary();
-            Function<HttpBeginExFW, MessageConsumer> correlation = factory.correlations.remove(connectReplyId);
-            if (correlation != null)
-            {
-                factory.router.clearThrottle(connectReplyId);
-            }
-            requestGroup.onNonCacheableResponse(acceptReplyId);
-            requestExpired = true;
-        }
-        else if (signalId == CACHE_ENTRY_ABORTED_SIGNAL)
-        {
-            if (this.payloadWritten >= 0)
-            {
-                factory.writer.doAbort(acceptReply,
-                                       acceptRouteId,
-                                       acceptReplyId,
-                                       signal.traceId());
-                requestGroup.unqueue(acceptReplyId);
-            }
-            else
-            {
-                send503RetryAfter();
-            }
-            cleanupRequestIfNecessary();
-            cacheEntry.setSubscribers(-1);
-        }
-        else if (signalId == INITIATE_REQUEST_SIGNAL)
-        {
-            doHttpBegin(getRequestHeaders());
-        }
+        factory.defaultCache.send304(ifNoneMatch,
+                                     prefer,
+                                     accept,
+                                     acceptRouteId,
+                                     acceptReplyId);
+        resetRequestTimeoutIfNecessary();
+        requestExpired = true;
```

Comment:
`requestExpired` needs to be renamed, since it no longer just means that the request expired, now it means the the response is closing, so let's rename to `responseClosing`.